test/spec/ol/feature.test.js

```diff
@@ -102,7 +102,7 @@ describe('ol.Feature', function() {
 
     it('returns null for null geometry (constructor)', function() {
```

Comment:
It would be nice to update this description to `it('returns undefined ...')`, but probably not worth it given the flakiness of the tests in Travis right now.